dokang/utils.py

```diff
@@ -7,10 +7,16 @@
 import os.path
 
 from dokang import api
+from dokang import compat
 
 
 def get_harvester(fqn):
-    module_fqn, function_fqn = fqn.rsplit(b'.', 1)
+    module_fqn, function_fqn = fqn.rsplit('.', 1)
+
+    if compat.PY2 and not isinstance(module_fqn, bytes):
+        module_fqn = module_fqn.encode()
+        function_fqn = function_fqn.encode()
```

Comment:
Yes, but it does not work for third party libraries : https://bugs.python.org/issue21720